src/client/dfuse/dfuse_main.c

```diff
@@ -349,6 +351,28 @@ show_help(char *name)
 	    name, DAOS_VERSION);
 }
 
+static int
+check_fd_mountpoint(const char *mountpoint)
+{
+	int fd  = -1;
+	int len = 0;
+
+	int res = sscanf(mountpoint, "/dev/fd/%u%n", &fd, &len);
+	if (res != 1) {
+		return -1;
+	}
+	if (len != strnlen(mountpoint, NAME_MAX)) {
+		return -1;
+	}
+
+	int fd_flags = fcntl(fd, F_GETFD);
```

Comment:
I'm slightly surprised this compiled as we normally keep variable declarations in a single block.  This is not critical however.
```suggestion
	int fd  = -1;
	int len = 0;
	int fd_flags;

	int res = sscanf(mountpoint, "/dev/fd/%u%n", &fd, &len);
	if (res != 1) {
		return -1;
	}
	if (len != strnlen(mountpoint, NAME_MAX)) {
		return -1;
	}

	fd_flags = fcntl(fd, F_GETFD);
```

Reply:
I moved all the declarations, even if it's not in the standard I want to be consistent with the rest of the file.